pkgs/servers/home-assistant/tests.nix

```diff
@@ -134,12 +134,7 @@ in lib.listToAttrs (map (component: lib.nameValuePair component (
     '';
 
     meta = old.meta // {
-      broken = lib.elem component [
-        # pinned version incompatible with urllib3>=2.0
-        "telegram_bot"
-        # depends on telegram_bot
-        "telegram"
-      ];
+      broken = lib.elem component [ ];
```

Comment:
@mweinelt Should I remove this line completely?

Reply:
No, I think it's a good reference.